### DISCUSSION TITLE
Add a `doctor` sub-command to dump support information to the console

### DISCUSSION THREAD

**Discussion by Lockszmith-GH** [Jip-Hop/jailmaker#103] [Ideas]
I think a _standarized_ format for support informaiton is a major boost for community support.
Presenting everything in a copy-pastable form directly from the consol into a GitHub discussion/issue format makes asking and providing support much easier.

The TrueNAS SCALE `cli` command allows gathering a lot of information in human readable form.

A scripted method of collecting this infomraiton, annonymizing it, and adding some GitHub specific markdown formatting (like `<details>` tag to auto-collapse long outputs, would take community support to the next level.

For examples of such commands, see a [recent discussion](../discussions/102#discussioncomment-8878238) I've been helping out with.

**Comment by Jip-Hop** [comment]
This could be helpful indeed. Until there'd be such a command I think I'll add an issue template on GitHub which asks for some debug info and links to https://github.com/Jip-Hop/jailmaker/discussions/135.

**Reply by Lockszmith-GH** [reply]
Might be a good idea to pin #135 to the top.
